content/concepts/server-api.md

```diff
@@ -8,9 +8,18 @@ tableOfContents: true
 
 ## API Overview
 
+By default, KES requires a valid certificate for any API call.
+Requests without a certificate fail with a TLS error.
+
+You can disable the requirement for a valid certificate when calling some endpoints in the KES [configuration file]({{< relref "tutorials/configuration.md#api-configuration" >}}).
+
+If any endpoint does not require a certificate, failed calls result in an HTTP error instead of a TLS error.
```

Comment:
This seems a bit ambiguous to me, would something like this be accurate:
> Endpoints that do not require a certificate return an HTTP error for a failed call instead of a TLS error.

I suspect other (certificate required) endpoints still return TLS errors on failure.